src/app/download/download.component.ts

```diff
@@ -0,0 +1,35 @@
+import {Component} from '@angular/core';
+import {AdAuthoringService} from '../ad-authoring/ad-authoring.service';
+import {Observable} from 'rxjs';
+import {map} from 'rxjs/operators';
+
+@Component({
+  selector: 'app-download',
+  templateUrl: './download.component.html',
+  styleUrls: ['./download.component.scss'],
+})
+export class DownloadComponent {
+  fileUrl;
+  ampHtmlObs: Observable<string>;
```

Comment:
readonly

Reply:
Removed the observable altogether because it's not needed here.